portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,201 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+/* Set up the content area grid */
+
+.content {
+  display: grid;
+  grid-template-rows: 55px 900px auto 270px;
+  grid-template-columns: auto;
+  grid-template-areas:
+    "header"
+    "intro"
+    "about"
+    "footer";
+}
+
+.row {
+  display: flex;
+}
+
+.column {
+  flex: 50%;
+}
+
+/* Reset default html margins and paddings */
+
+html, body { 
+  margin: 0; 
+  padding: 0;
+}
+
+html { 
+  scroll-behavior: smooth;
+}
+
+/* Style the text selection color */
+ 
+::selection {
+    color: rgb(80, 80, 80); 
+    background: rgba(230, 175, 184, .822);
+}
+
+::-moz-selection { 
+    color: rgb(80, 80, 80);
+    background: rgba(230, 175, 184, .822);
+}
+
+/* Style different headings and paragraph text */
+
+h1 { 
+  font-family: 'Poppins';
+  font-size: 90px; 
+  font-style: bold;
+  font-weight: 700; 
+  line-height: 26.4px; 
+} 
+
+h2 {
+  font-family: 'Poppins'; 
+  font-size: 24px;  
+  font-variant: normal;  
+  font-weight: 400;
+  line-height: 23px; 
+}
+
+h3 { 
+  font-family: 'Montserrat'; 
+  font-size: 24px; 
+  font-style: normal; 
+  font-variant: small-caps; 
+  font-weight: 400;  
+  margin: 0px;
+}
+
+p {
+  font-family: 'Montserrat'; 
+  font-size: 15px; 
+  font-style: normal; 
+  font-variant: normal; 
+  font-weight: 400; 
+}
+
+
+/* Style the header section */
+
+.header { 
+  background-color: rgb(255, 177, 104);
+  box-shadow: 0px 5px 39px 2px rgba(69,69,69,0.71);
+  -moz-box-shadow: 0px 5px 39px 2px rgba(69,69,69,0.71);
+  -webkit-box-shadow: 0px 5px 39px 2px rgba(69,69,69,0.71);
+  color: white;
+  grid-area: header;
+  overflow: hidden;
+  position: fixed; 
+  top: 0; 
+  width: 100%; 
+}
+
+nav {
+    margin-left: 150px;
+    margin-top: 20px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+/* Style the links inside the navigation bar */
+
+nav a {
+  color: white; 
+  float: left; 
+  font-family: "Poppins"; 
+  font-size: 17px;
+  font-weight: 300; 
+  padding: 0 20px 20px 20px;
```

Comment:
In case you didn't know, padding and margin can actually collapse the top/bottom and left/right if they use the same measurement, so here you can do:
`padding: 0 20px 20px;`
Not really the most useful in this scenario, but just a helpful hint